### DISCUSSION TITLE
404 on every deployment

### DISCUSSION THREAD

**Discussion by yigalavi** [vercel/community#6360] [Help]
### Summary

Hi, just started now, got the cf worker synced easily from github, but I get 404 white page for any deployment.
Even after deploying the simplest nodejs hello world template, it just says:
404: NOT_FOUND
Code: NOT_FOUND
ID: fra1::xl7n7-1711960000792-611ed7a6440d

### Steps to Reproduce

Just browse to https://player-text-vercel.vercel.app/ the code is just the nodejs hello world template.

**Comment by saradhapreetha** [comment]
Hi, please check you vercel.json file.
{ "version": 2,
"builds":[{"src":"server.js","use":"@vercel/node"}]}
adding this helps if your root file name is not index.js which is default/
